sycl/test-e2e/NonUniformGroups/is_fixed_topology.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %{build} -fsyntax-only -o %t.out
+// RUN: %{build} -fsyntax-only -Wno-error=unused-command-line-argument -o %t.out
```

Comment:
Once again, let's move the test instead of modifying it. This one I'm even aware of and have it my backlog, but who knows when I get to it